src/handlers/itemStore.ts

```diff
@@ -2,8 +2,8 @@ import { Subscription } from "nats";
 
 import { jsonCodec, NatsHandler } from "../nats";
 import { getClient } from "../elasticSearch";
-import { ELASTICSEARCH_INDEX_NAME } from "../config";
-import { JSONItem } from "../item";
+import { ELASTICSEARCH_INDEX_NAME, ELASTICSEARCH_INVENTORY_INDEX_NAME } from "../config";
+import { JSONItem, JSONInventoryItem } from "../item";
```

Comment:
let's rename JSONInventoryItem to `InventoryItem` just like we have `WarehouseItem`